ui/app/controllers/vault/cluster/auth.js

```diff
@@ -62,9 +62,7 @@ export default Controller.extend({
     }
     transition.followRedirects().then(() => {
       if (isRoot) {
-        this.flashMessages.warning(
-          'You have logged in with a root token. As a security precaution, this root token will not be stored by your browser and you will need to re-authenticate after the window is closed or refreshed.'
-        );
+        this.auth.set('showRootTokenWarning', true);
```

Comment:
I wonder if we should skip this if we're testing 🤔 
(probably a better way to write this, but just as a suggestion)
```suggestion
        if (Ember.testing) return
        this.auth.set('showRootTokenWarning', true);
```

Reply:
that's a great idea! i forgot we can skip specific code blocks in testing. thanks claire!

Reply:
It's hard for me to see in the diff, but did you implement Claire's suggestion?